app/routes/video_routes.py

```diff
@@ -175,24 +203,72 @@ def stream_video(video_id: str, db: Session = Depends(get_db)):
     raise HTTPException(status_code=404, detail="Video not found.")
 
 
+@router.get("/recording/transcript/{video_id}")
+def get_transcript(video_id: str, db: Session = Depends(get_db)):
+    """
+    Get the transcript for a video by its video ID.
+
+    Parameters:
+        video_id (int): The ID of the video to be streamed.
+        db (Session, optional): The database session. Defaults to the
+            result of the get_db function.
+
+    Returns:
+        FileResponse: The file response containing the video stream.
+
+    Raises:
+        HTTPException: If the video is not found.
+    """
+    video = db.query(Video).filter(Video.id == video_id).first()
+    db.close()
+
+    if video:
+        return FileResponse(video.transcript_location, media_type="text/plain")
+    raise HTTPException(status_code=404, detail="Video not found.")
```

Comment:
Let's use early return pattern here
```suggestion
    if not video:
         raise HTTPException(status_code=404, detail="Video not found.")
    return FileResponse(video.transcript_location, media_type="text/plain")
    
```